packages/attestation-service/src/requestHandlers/status.ts

```diff
@@ -1,8 +1,9 @@
 import { AttestationServiceStatusResponseType, SignatureType } from '@celo/utils/lib/io'
 import express from 'express'
 import * as t from 'io-ts'
-import { kit } from '../db'
-import { getAccountAddress, getAttestationSignerAddress } from '../env'
+import { version as VERSION } from '../../package.json'
```

Comment:
You could also run it via a yarn command and then this gets put into an env var:
https://github.com/celo-org/celo-monorepo/blob/master/packages/phone-number-privacy/signer/src/config.ts#L3

Reply:
This just saved me!